presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestDbManagerSpecProvider.java

```diff
@@ -0,0 +1,138 @@
+        dao.insertResourceGroup(1, "prod_global", "10MB", 1000, 100, 100, "weighted", null, true, "1h", "1d", null, prodEnvironment);
+        dao.insertResourceGroup(2, "dev_global", "1MB", 1000, 100, 100, "weighted", null, true, "1h", "1d", null, devEnvironment);
+        dao.insertSelector(1, 1, ".*prod_user.*", null, null, null, null);
+        dao.insertSelector(2, 2, ".*dev_user.*", null, null, null, null);
+
+        // check the prod configuration
+        DbManagerSpecProvider dbManagerSpecProvider = new DbManagerSpecProvider(daoProvider.get(), prodEnvironment, new ReloadingResourceGroupConfig());
+        ReloadingManagerSpec reloadingManagerSpec = dbManagerSpecProvider.getSpec();
+        assertEquals(reloadingManagerSpec.getResourceGroups().size(), 1);
+        assertEquals(reloadingManagerSpec.getManagerSpec().getRootGroups().size(), 1);
+        assertEquals(reloadingManagerSpec.getManagerSpec().getSelectors().size(), 1);
+        SelectorSpec prodSelector = reloadingManagerSpec.getManagerSpec().getSelectors().get(0);
+        assertEquals(prodSelector.getGroup().toString(), "prod_global");
+
+        // check the dev configuration
+        dbManagerSpecProvider = new DbManagerSpecProvider(daoProvider.get(), devEnvironment, new ReloadingResourceGroupConfig());
+        reloadingManagerSpec = dbManagerSpecProvider.getSpec();
+        assertEquals(reloadingManagerSpec.getResourceGroups().size(), 1);
+        assertEquals(reloadingManagerSpec.getManagerSpec().getRootGroups().size(), 1);
+        assertEquals(reloadingManagerSpec.getManagerSpec().getSelectors().size(), 1);
+        prodSelector = reloadingManagerSpec.getManagerSpec().getSelectors().get(0);
+        assertEquals(prodSelector.getGroup().toString(), "dev_global");
+    }
+
+    @Test
+    public void testExactMatchSelector()
+    {
+        H2DaoProvider daoProvider = setup("test_exact_match_selector");
+        H2ResourceGroupsDao dao = daoProvider.get();
+        dao.createResourceGroupsGlobalPropertiesTable();
+        dao.createResourceGroupsTable();
+        dao.createSelectorsTable();
+        dao.createExactMatchSelectorsTable();
+        dao.insertResourceGroup(1, "global", "1MB", 1000, 100, 100, "weighted", null, true, "1h", "1d", null, ENVIRONMENT);
+        dao.insertResourceGroup(2, "sub", "2MB", 4, 3, 3, null, 5, null, null, null, 1L, ENVIRONMENT);
+        dao.insertSelector(2, 1, null, null, null, null, null);
+        dao.insertResourceGroupsGlobalProperties("cpu_quota_period", "1h");
+
+        DbManagerSpecProvider dbManagerSpecProvider = new DbManagerSpecProvider(daoProvider.get(), ENVIRONMENT, new ReloadingResourceGroupConfig());
+        assertEquals(dbManagerSpecProvider.getSpec().getResourceGroups().size(), 2);
+        assertEquals(dbManagerSpecProvider.getSpec().getManagerSpec().getSelectors().size(), 1);
+        assertTrue(dbManagerSpecProvider.getExactMatchSelectors().get(0) instanceof DbSourceExactMatchSelector);
```

Comment:
This is not testing anything specific to exact match selectors. We can skip this test altogether